src/app/public/modules/file-attachment/file-attachment.component.ts

```diff
@@ -78,6 +78,9 @@ export class SkyFileAttachmentComponent implements ControlValueAccessor, AfterVi
   @Input()
   public acceptedTypes: string;
 
+  @Input()
```

Comment:
For the `ariaRequired` pull request (forms), is there ever a situation where we’d want the field to be "required" but not `aria-required`? I think that the source of truth should be the `required` input, and the component should automatically set the `aria-required` attribute based on that? https://github.com/blackbaud/skyux-forms/blob/master/src/app/public/modules/file-attachment/file-attachment.component.ts#L108